src/UiPath.Workflow/UiPath.Workflow.csproj

```diff
@@ -30,6 +30,9 @@
         <ItemGroup>
             <BuildOutputInPackage Include="$(OutputPath)Microsoft.CodeAnalysis.VisualBasic.Scripting.*"/>
         </ItemGroup>
+		<ItemGroup Condition="'$(TargetFramework)' == 'net6.0' AND '$(Configuration)'=='Debug'">
+			<BuildOutputInPackage Include="$(OutputPath)System.Xaml.dll"/>
+		</ItemGroup>
```

Comment:
Not mandatory, but helps with debugging in Studio.
This only applies on debug build.